strfry/plugins/policies.ts

```diff
@@ -12,16 +12,20 @@ import nosPolicy from "./nos_policy.ts";
 
 const localhost = "127.0.0.1";
 const eventsIp = await getEventsIp();
+
 const one_minute = 60 * 1000;
-const two_days = 2 * 24 * 60 * one_minute;
+const one_hour = 60 * one_minute;
+const one_day = 24 * one_hour;
+const two_days = 2 * one_day;
 
 // Policies that reject faster should be at the top. So synchronous policies should be at the top.
 const policies = [
   nosPolicy,
   [hellthreadPolicy, { limit: 100 }],
 
   // Async policies
-  [antiDuplicationPolicy, { ttl: 60000, minLength: 50 }],
+  // Let's test with one day, if it's too much we can reduce it.
+  [antiDuplicationPolicy, { ttl: one_day, minLength: 30 }],
```

Comment:
The other important change I added in this commit that we need to test and see how it behaves. Hashes of all event contents are now remember for a day if they contain more than 30 words. Not sure how effective this could be but it's just a small config change.